src/cmd/report.go

```diff
@@ -111,11 +112,11 @@ type githubRelease struct {
 	Assets []githubReleaseAsset
 }
 
-func fetchGitHubData(repo string) {
+func reportGitHubData(repo string) {
 
 	resp, err := http.Get("https://api.github.com/repos/" + repo + "/releases")
 	if err != nil {
-		log.Fatal("Error: Failed to fetch GitHub data.")
+		log.Fatal("Error: Failed to report GitHub data.")
```

Comment:
These functions should still be fetch as they is what they are doing. The command is reporting, not these functions.